mod_reforged/hooks/config/character_names.nut

```diff
@@ -131,6 +132,7 @@
 	"Coldsteel",
 	"Ironborn"
 ];
+
```

Comment:
This extra line has nothing to do with the changes in this PR and should not be included.